fvm/accounts_test.go

```diff
@@ -1274,7 +1274,7 @@ func TestAccountBalanceFields(t *testing.T) {
 				assert.NoError(t, err)
 				assert.NoError(t, script.Err)
 
-				assert.Equal(t, cadence.UFix64(9999_5030), script.Value)
+				assert.Equal(t, cadence.UFix64(9999_5020), script.Value)
```

Comment:
Hey @ramtinms this value changed after we deployed some extra contracts during execution state bootstrapping in this PR - is that expected and is it OK to update this expected value?

Reply:
yeah should be fine, it makes sense if we have deployed more contracts